src/__tests__/RenderParentDashboard.test.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
-import { render, screen, cleanup } from '@testing-library/react';
+import { render, screen, cleanup, fireEvent } from '@testing-library/react';
```

Comment:
Needed fireEvent in order to be able to fire a mouse click on the hamburger button.

Reply:
I notice this file comes first on the PR, did you write these tests before writing the UI and functionality of the elements you were testing? 

Correct me if I am wrong but it looks like these tests only check if the UI is rendered correctly to the page, is there a reason you didn't also test the functionality?

Reply:
I am utilizing some TDD here, so I decided to write the test suite prior to the component itself.